### PR TITLE
Mountain animation

### PR DESCRIPTION
**Pull request by sofiapoh**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -16,16 +16,8 @@
   <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
   <link href="https://fonts.googleapis.com/css?family=Amatic+SC|Open+Sans+Condensed:300|Raleway:500" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <!-- <meta name="viewport" content="width=device-width, initial-scale=1"/> -->
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
```
Comment (gabrielperales) [inline src/index.html:19]: why there are two `meta=viewport`?

**Review by gabrielperales — APPROVED** [review]

**Comment by gabrielperales** [comment]
Seems good to me :)

File: src/index.js [hunk 2]
```diff
@@ -64,9 +64,8 @@ window.addEventListener('load', () => setTimeout(() => {
     screenfull.request(document.body);
   });
 
-
-  if (breathingPageVisited && location.hash !== '#breathe') {
-    location.hash = '#breathe';
+  if (!breathingPageVisited) {
```
Comment (gabrielperales) [inline src/index.js:67]: this two lines were ok. Now the app is not going to the breathing page after the first visit.

**Review by gabrielperales — CHANGES_REQUESTED** [review]
it should redirect to '#breathe'

File: src/index.html (hunk 1 above)
Comment (sofiapoh) [reply src/index.html:19]: taken care of!

**Review by gabrielperales — APPROVED** [review]

**Pull request merged**